reference/customers.v3.yml

```diff
@@ -712,6 +713,62 @@ paths:
           headers: {}
       tags:
         - Customer Addresses
+  /customers/validate-credentials:
+    post:
+      tags:
+        - Customer Validate Credentials
+      description: Validate a customer credentials - This endpoint has special rate limiting protections to protect against abuse.
+      summary: Validate a customer credentials
+      operationId: CustomerValidateCredentials
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ValidateCustomerCredentialsObject'
+            examples:
+              data:
+                value:
+                  email: 'foo@bar.com'
+                  password: 'password'
+                  channel_id: 1
+      responses:
+        '200':
+          description: Returns if the customer credentials provided are valid or not
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ValidateCustomerCredentialsResponseObject'
+              examples:
+                data:
+                  value:
+                    is_valid: true
+        '429':
+          description: |
+            This endpoint has special rate limiting protections to protect against abuse.
```

Comment:
`Allowed number of requests exceeded` ?

Reply:
I wasn't sure how to articulate it so I've reused the generic one we have for instance for the actual V2 endpoint.